Mathlib/Algebra/BigOperators/Group/Finset.lean

```diff
@@ -1316,11 +1316,17 @@ test on the index and whose alternative is `0` has value either the term at that
 
 The difference with `Finset.sum_ite_eq` is that the arguments to `Eq` are swapped."]
 theorem prod_ite_eq' [DecidableEq α] (s : Finset α) (a : α) (b : α → β) :
-    (∏ x ∈ s, ite (x = a) (b x) 1) = ite (a ∈ s) (b a) 1 :=
+    (∏ x ∈ s, if x = a then b x else 1) = if a ∈ s then b a else 1 :=
   prod_dite_eq' s a fun x _ => b x
 #align finset.prod_ite_eq' Finset.prod_ite_eq'
 #align finset.sum_ite_eq' Finset.sum_ite_eq'
 
+@[to_additive]
+theorem prod_ite_eq_iff [DecidableEq α] (s : Finset α) (a : α) (b : α → β)
+    {p : α → Prop} [DecidablePred p] (h : ∀ x ∈ s, p x ↔ a = x) :
+    (∏ x in s, if p x then b x else 1) = if a ∈ s then b a else 1 :=
+  (prod_ite_eq s a b).symm ▸ prod_congr rfl (fun a ha => if_congr (h a ha) rfl rfl)
```

Comment:
I don't think we want that lemma? It's just a `prod_congr` away from an existing one

Reply:
IIRC I copied it from `Algebra/BigOperators/Finsupp.lean`. I will remove it

Reply:
Seems like you also added the `Algebra.BigOperators.Finsupp` one 🤔

Reply:
lmfao sorry